apparmor.d/groups/virt/cni-portmap

```diff
@@ -0,0 +1,14 @@
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{opt/,}{cni/,}bin/portmap
```

Comment:
Any justification of `/{opt/,}{cni/,}bin/` instead of `/opt/cni/bin/` ?

Reply:
I honestly don't know if other CNI providers like Flannel or Cilicium install the same commands in the same folders. I figured I'd cover most options.

Reply:
The `cni-plugins` package seems to install it in `/opt/cni/bin/`, so it should be enough.
Plus: `/{opt/,}{cni/,}bin/` allow stuff like:
* `/cni/bin/` No way to allow this 
* `/opt/bin/` that does not exist too.

Reply:
Sounds good, I've pushed another commit to fix this.